mods/flowers/init.lua

```diff
@@ -236,7 +240,8 @@ function flowers.mushroom_spread(pos, node)
 	end
 	local pos2 = positions[math.random(#positions)]
 	pos2.y = pos2.y + 1
-	if minetest.get_node_light(pos2, 0.5) <= 3 then
+	if minetest.get_node_light(pos, 0.5) <= 3 and
+			minetest.get_node_light(pos2, 0.5) <= 3 then
```

Comment:
Unrelated changes here as well, please undo.

Reply:
THX and will depollute...